internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -283,6 +283,18 @@ Resources:
                     # Required otherwise ContentType won't be applied.
                     # See https://github.com/aws/aws-sdk-js/issues/1092 for more.
                     MetadataDirective: "REPLACE"
+          InvalidateCache:
+            Type: Task
+            End: true
+            Resource: arn:aws:states:::aws-sdk:cloudfront:createInvalidation
+            Parameters:
+              DistributionId: !Ref CloudFrontDistribution
+              InvalidationBatch:
+                CallerReference.$: States.UUID()
+                Paths:
+                  Quantity: 1
```

Comment:
Is the quantity here essentially `len(Items)`?

Reply:
Yes. Do you think I should use `Fn::Length:` instead? I think in this case it may be more straightforward to just hard code the number bc we're always using the wildcard??

Reply:
Yeah if it works i guess i'll make more sense and explains itself

Reply:
Ah, it's an unsupported function. :(